js/index.d.ts

```diff
@@ -0,0 +1,302 @@
+declare module 'react-native-mparticle';
+export default MParticle;
```

Comment:
**issue:** i think you need to change to lines 1 and 2 to:
```
declare module 'react-native-mparticle' {
  export default MParticle;
}
```

As currently implemented the module is declared, but is empty. This results in the same error as before: `Could not find a declaration file for module 'react-native-mparticle'`